include/circt/Dialect/Ibis/IbisOps.td

```diff
@@ -189,6 +190,29 @@ def ReturnOp : IbisOp<"return", [
   ];
 }
 
+def ScheduleOp : IbisOp<"schedule", [
+  HasParent<"MethodOp">,
+  SingleBlock,
+  NoRegionArguments,
+  NoTerminator
+]> {
+  let summary = "Ibis scheduling constraint block";
+  let description = [{
+    The `ibis.schedule` operation defines a block wherein a group of operations
+    can be placed wherein a limited number of Ibis threads can execute at once.
+    The operation is isolated from above and has no return types. References
```

Comment:
I don't see the isolated from above trait... did you mean *not* isolated from above?

Reply:
I did mean **not** - woops.